WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -75,15 +75,29 @@ private extension CardReaderSettingsConnectedViewController {
     ///
     func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
         switch cell {
-        case let cell as TitleTableViewCell where row == .temporaryHeader:
-            configureHeader(cell: cell)
+        case let cell as ConnectedReaderTableViewCell where row == .connectedReader:
+            configureConnectedReader(cell: cell)
+        case let cell as ButtonTableViewCell where row == .disconnectButton:
+            configureButton(cell: cell)
         default:
             fatalError()
         }
     }
 
-    private func configureHeader(cell: TitleTableViewCell) {
-        cell.titleLabel?.text = Localization.temporaryHeader
+    private func configureConnectedReader(cell: ConnectedReaderTableViewCell) {
+        guard let serialNumber = viewModel?.connectedReaderSerialNumber,
```

Comment:
I'm just curious, what would happen if we assigned those values to the labels text property directly? `.text` is optional, so setting it to nil will just clear the label (iirc)